blog/_posts/2017-09-15-d-for-diamonds.md

```diff
@@ -0,0 +1,53 @@
+
+## What we built
+
+Our project was based on creating plugins to let the admins back up the information of the forum in some cloud provider.
+
+We didn't know how long it would take us to build one plugin. I thought we would have one or two at the end of the summer, but we ended up [enabling 4 possibilities](https://meta.discourse.org/t/new-plugins-for-backups/68018) as well as a base class plugin from which all of them inherit (Google Drive, Nextcloud - Jan from Nextcloud left a message in our repo suggesting it, we were super happy to get it and built it right away! - Box, and a new version of the existent backups to Dropbox by [falco](https://github.com/xfalcox)). After this, our mentor [eviltrout](https://eviltrout.com/) found another cool task to give us: downloading these files directly from Discourse and send an email to the user when it's done, which we are on the way of building.
+
+![Team berlindiamonds](/img/blog/2017/2017-09-15-berlindiamonds.png)
+<div class="image-credits">Github screenshot</div>
+
+We've learnt so many things this summer: inheritance, injecting code through class_eval and then through event listeners, git git git, a lot of interaction with APIs, the power of gems, how they're built, code structure, the "super" keyword, memoization, triggering Jobs, executing stuff in the console, debugging with puts, byebug, Rails.logger.debug, some testing, drawing routes from the outside, Ember basics and the importance of proper documentation!
+
+![Team berlindiamonds](/img/blog/2017/2017-09-15-berlindiamonds.jpg)
+<div class="image-credits">Absolventa</div>
+
+We also participated in a RG beginners workshop in August, Kaja is part of the organization and proposed me to coach. It was a great feeling to see so many women (around 40) curious to code and many coaches happy to share their skills.
+
+## A huge "thank you" to:
+
+The whole Rails Girls Summer of Code organization and supporters for giving life to this amazing project. This means the world to all newcomers and it fills the so important [tutorial/junior gap mentioned by Daniel Kehoe](https://books.google.de/books?id=tvazDQAAQBAJ&pg=PT83&lpg=PT83&dq=%22tutorial+gap%22+rails&source=bl&ots=zu4r8J1vpx&sig=62dYwycJiCjwiDrNAFAkb6RzD7s&hl=en&sa=X&ved=0ahUKEwjZ4fzs7JzWAhXKUlAKHaRrBOQQ6AEILjAB#v=onepage&q=%22tutorial%20gap%22%20rails&f=false).
+
+Our in and offsite coaches for volunteering in this life-changing project. Robin, Markus and Carsten at Absolventa who've answered passionately so many questions the moment we've asked them, and took the time to guide us and use the whiteboard to explain the universe of Rails. To Rojo, the most active online coach who've challenged us with his answers to find the solution by ourselves, chapeau! To Robert, who's been there in our team events and found some time to help out online too.
+
+![Team berlindiamonds](/img/blog/2017/2017-09-15-berlindiamonds-team.png)
```

Comment:
I think you meant to you berlindiamonds_w_markus image here. Can you fix that as well?

Reply:
@nerdyvaishali fixed :)

Reply:
@jen-v Thanks :)